src/util-time.c

```diff
@@ -360,7 +365,11 @@ void CreateTimeString (const struct timeval *ts, char *str, size_t size)
     int lru = 1 - mru;
     int mru_seconds = time - last_local_time[mru];
     int lru_seconds = time - last_local_time[lru];
-    if (mru_seconds >= 0 && mru_seconds <= 59) {
+    if (last_local_time[mru]==0 && last_local_time[lru]==0) {
+        /* First time here, update both caches */
+        seconds = UpdateCachedTime(mru, time);
+        seconds = UpdateCachedTime(lru, time);
```

Comment:
this looks suspicious... why overwrite seconds?

Reply:
It shouldn't cause any trouble as it will run UpdateCacheTime with the same value (UpdateCacheTime will return the seconds from SCLocalTime). I can remove it if that would make it look clearer.